client/src/index.js

```diff
@@ -104,16 +75,14 @@ ReactDOM.render(
           render={(props) => (
             <IndexPage
               {...props}
-              token={token}
-              isAuthenticated={isAuthenticated}
-              storageKeys={storageKeys}
             />
           )}
         />
         <Route path='/icons' render={(props) => <NucleoIcons {...props} />} />
-        {/* <Redirect to="/" /> */}
+        <Redirect to="/" />
```

Comment:
Instead of a redirect as the last route, should we have a 404 fallback?

Reply:
:+1: